tests/utils/GraphicalUnitTester.py

```diff
@@ -117,6 +118,14 @@ def _show_diff_helper(self, frame_data, expected_frame_data):
 
     def test(self, show_diff=False):
         """Compare pre-rendered frame to the frame rendered during the test."""
+
+        # If the class has the pytest_skip attribute, mark this test as skipped.
+        try:
+            if self.scene.pytest_skip:
+                return pytest.skip("Scene flagged with skipped test.")
```

Comment:
**Nitpick** : Could you make the scene display instead of "Scene"?

Reply:
I'll be removing this feature entirely as it's really just for my development process.